src/fileActions.jsx

```diff
@@ -436,16 +438,18 @@ const EditPermissionsModal = ({ selected, path }) => {
     );
 };
 
-const setDirectoryName = (val, setName, setNameError, setErrorMessage) => {
+const setDirectoryName = (val, setName, setNameError, setErrorMessage, cwdInfo) => {
```

Comment:
This function should be renamed now.

Note also, the mkdir PR stops using this function (since it creates its own copy in a separate file).

Reply:
I mean, this function has always had the wrong name (since it was used from the rename dialog as well) but you've fixed the messages here, but not the function name...